src/main/resources/api.yaml

```diff
@@ -592,22 +603,36 @@ components:
     SprintSummaryDto:
       type: object
       properties:
+        sprint:
+          $ref: '#/components/schemas/SprintDto'
         membersAvailability:
           type: array
           items:
             $ref: '#/components/schemas/UserAvailabilityDto'
           description: List of members' availability for given sprint
-        totalAvailableTime:
+        prevAverageSprintCoefficient:
+          type: number
+          format: double
+        currentAverageSprintCoefficient:
+          type: number
+          format: double
+        timeToAssign:
+          type: integer
+          format: int64
+          description: In minutes
+        totalDeclaredTime:
+          type: integer
+          format: int64
+          description: In minutes
+        totalNeededTime:
           type: integer
           format: int64
           description: In minutes
-        sprintCoefficient:
-          type: number
-          format: double
       required:
+        - sprint
         - membersAvailability
-        - totalAvailableTime
-        - sprintCoefficient
+        - prevAverageSprintCoefficient
+        - totalDeclaredTime
```

Comment:
Jaki jest algorytm za akurat `prevAverageSprintCoefficient` i `totalDeclaredTime`  jako required?

Reply:
poprzedni wspolczynnik musi byc bo powinien on byc znany zaraz po zamknieciu sprintu (z integracja z jiry mozliwie pozniej). Total declared time czemu nie, defaultowo 0

Reply:
to nie jest stricte to co musi byc wysylane, ale raczej to co powinno byc zawsze znane.
moge wyrzucic z required jeśli nie chcemy tego enforsować

Reply:
Nie no, może być - to jest i tak DTO tylko do GETów - po prostu byłam ciekawa, dlaczego akurat te dwa z miliona pól